httpx/client_test.go

```diff
@@ -72,6 +108,29 @@ func TestFaultTolerance(t *testing.T) {
 	assert.Equal(200, resp.StatusCode)
 }
 
+func TestInstrumentRequestDurationMetric(t *testing.T) {
+	assert := assert.New(t)
+
+	timer := &TimerMock{TagsMap: make(map[string]interface{})}
+	metricsInstrument := &MetricsInstrument{timerMock: timer}
+	customTag := metrics.Tag{Key: "tag", Value: "test"}
```

Comment:
As a general rule I think is better to put the `"tag"` and `"test"` values in vars so you can use them in L126 and L127 without duplication.